askama_derive/src/generator.rs

```diff
@@ -237,11 +236,46 @@ fn find_used_templates(
                 _ => {}
             }
         }
-        map.insert(path, source);
+        map.insert(path, parsed);
     }
     Ok(())
 }
 
+mod _parsed {
+    use std::mem;
+
+    use crate::config::Syntax;
+    use crate::parser::{parse, Node};
+    use crate::CompileError;
+
+    pub(super) struct Parsed<'config> {
```

Comment:
Does this actually need a named lifetime? I think `Node<'config>` can be `Node<'static>`. Parsed::nodes() reduces the lifetime to `Node<'self>` as it should be.

Reply:
The compiler was complaining that the lifetime of the syntax has to outlive the `Node`'s lifetime, which seems reasonable?

Reply:
That's because of `parse<'a>(src: &'a str, syntax: &'a Syntax<'_>)`. `&'a Syntax<'_>` should be `&Syntax<'_>`: https://github.com/djc/askama/compare/parse-once...Kijewski:askama:parse-once

Reply:
Nice, thanks!